### DISCUSSION TITLE
Computing confidence intervals

### DISCUSSION THREAD

**Discussion by Percival24** [nlmixr2/nlmixr2#121] [Q&A] [answered]
Hi :) 
I am trying to understand how nlmixr2 proceeds to compute the confidence intervals after the estimation run... I tried to use the output from the fit but I am not able to find back the confidence intervals that it shows me... :/ Any hint would be more than welcome :)
Best regards, 
Percival

**Comment by mattfidler** [comment, marked as answer]
It calculates the confidence interval on the untransformed space and then back-transforms it.

**Reply by Percival24** [reply]
Using exp(quantile(fit$parHist$tcl,0.5)) I can find back the estimation of cl as shown in fit$parFixed. But I can't do the same game for the confidence intervals... Sorry, my question is more related to my lack of knowledge about pk models than anything else... I am more used to MCMC procedures but, obviously, FOCEi is not an MCMC procedure...

**Reply by mattfidler** [reply]
It actually dosen't matter if it is a `mcmc` procedure, the confidence intervals are calculated based on the standard error and the Z-score for the requested confidence band.  Giving the same number for the `cl` means that it is symmetric, and the percentiles match the confidence bands calculated through the standard normal stats confidence interval

**Comment by Percival24** [comment]
Thanks @mattfidler ! He can find back my confidence intervals ;)

**Comment by umachkaalex** [comment]
Good day.

Is there a way to get confidence intervals for the components of the variance-covariance matrix of random effects (similar to "CI lower" and "CI upper" in fit$parFixedDf for fixed effects.)? Specifically, in "focei" regime (if this is important).

**Reply by mattfidler** [reply]
Not currently.

**Reply by umachkaalex** [reply]
Thanks
